custom_components/multiscrape/util.py

```diff
@@ -19,14 +17,14 @@ def create_renderer(hass, value_template):
     else:
         value_template.hass = hass
 
-    def _render(value: Any = None, parse_result=False):
+    def _render(variables: dict = {}, parse_result=False):
         try:
-            return value_template.async_render({"value": value}, parse_result)
+            return value_template.async_render(variables, parse_result)
         except TemplateError:
             _LOGGER.exception(
-                "Error rendering template: %s with value %s", value_template, value
+                "Error rendering template: %s with variables %s", value_template, variables
             )
-            return value
+            return None
```

Comment:
@danieldotnl, I'm returning None now. Is that ok?

Reply:
I think we should re-raise the exception as we are in trouble anyway when this happens.

Reply:
Makes sense.